src/main/webapp/consentGroupReview/ConsentGroupReview.js

```diff
@@ -349,11 +349,9 @@ class ConsentGroupReview extends Component {
   };
 
   isCurrentUserAdmin() {
-    User.isCurrentUserAdmin(this.props.isAdminUrl).then(
-      resp => {
-        this.setState({ isAdmin: resp.data.isAdmin });
-      }
-    );
+    isCurrentUserAdmin().then(response => {
```

Comment:
Ok, I see where this is coming from now `import { isCurrentUserAdmin, isEmpty } from "../util/Utils";` ... and the fact the parent method name is the same as the imported function is very confusing. Please rename one or the other so it doesn't look like a recursive call.